crates/core/src/constants.rs

```diff
@@ -1 +1,3 @@
-pub use grit_core_patterns::constants::DEFAULT_FILE_NAME;
+pub use grit_pattern_matcher::constants::DEFAULT_FILE_NAME;
+
+pub const MAX_FILE_SIZE: usize = 1_000_000;
```

Comment:
This constant was moved from `grit-pattern-matcher`'s constants, since it's only used by `marzano-core`.